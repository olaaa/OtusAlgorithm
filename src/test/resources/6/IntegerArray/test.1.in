29
28
77
52
80
14
25
58
18
94
22
97
42
50
68
57
5
73
9
95
61
54
27
59
69
63
21
83
60
9
86
78
63
9
79
27
61
81
64
98
46
9
31
93
84
86
20
1
74
13
97
92
30
89
2
67
15
98
86
93
32
45
30
15
69
8
18
51
20
96
49
60
87
72
64
72
68
38
28
72
97
73
71
85
63
13
6
88
68
10
25
29
6
15
43
53
22
22
48
58