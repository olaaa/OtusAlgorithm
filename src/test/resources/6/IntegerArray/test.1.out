[1, 2, 5, 6, 6, 8, 9, 9, 9, 9, 10, 13, 13, 14, 15, 15, 15, 18, 18, 20, 20, 21, 22, 22, 22, 25, 25, 27, 27, 28, 28, 29, 29, 30, 30, 31, 32, 38, 42, 43, 45, 46, 48, 49, 50, 51, 52, 53, 54, 57, 58, 58, 59, 60, 60, 61, 61, 63, 63, 63, 64, 64, 67, 68, 68, 68, 69, 69, 71, 72, 72, 72, 73, 73, 74, 77, 78, 79, 80, 81, 83, 84, 85, 86, 86, 86, 87, 88, 89, 92, 93, 93, 94, 95, 96, 97, 97, 97, 98, 98]